### PR TITLE
fix: updating tabbar home icon for sample app

### PR DESCRIPTION
**Pull request by rajamatt**
GitHub Issue (If applicable): closes unoplatform/uno#11059

## PR Type
What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?
- Home icon is visible on Wasm/Skia.
- Home icon is appearing correctly on iOS/Android.

## PR Checklist
Please check if your PR fulfills the following requirements:
- [x] Tested on iOS.
- [x] Tested on Wasm.
- [x] Tested on Android.
- [x] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

## Other information
Material TabBar: material tabbar
Cupertino TabBar: cupertino-tabbar

### CODE REVIEW

File: Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml [hunk 1]
```diff
@@ -33,7 +33,7 @@
 			<Grid x:Name="Home" Visibility="Collapsed">
 				<Border Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 					<StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
-						<FontIcon Glyph="&#xE80F;" />
+						<FontIcon Glyph="&#xE10F;" />
```
Comment (Arieldelossantos) [inline Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml:36]: can we set  Glyph as staticResource on the top of the page to be reausable??(this comment applies for the rest of the page you did the update :p)

**Review by kazo0 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml (hunk 1 above)
Comment (Xiaoy312) [reply Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml:36]: You should be able to do that. FontIcon::Glyph is just a string.
But, validate on uwp and one of uno targets to be sure.

**Review by Xiaoy312 — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml (hunk 1 above)
Comment (rajamatt) [reply Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml]: @kazo0

Comment (kazo0) [reply Uno.Gallery/Uno.Gallery.UWP/Views/NestedPages/CupertinoBottomBarSampleNestedPage.xaml]: You can create a new file, like FontIcons.xaml or whatever makes sense, similar to the [Path.xaml](https://github.com/unoplatform/Uno.Gallery/blob/master/Uno.Gallery/Uno.Gallery.UWP/Views/Styles/Path.xaml)

**Pull request merged**